indolaw-parser/crawler/client_secrets.json

```diff
@@ -0,0 +1 @@
+{"web":{"client_id":"911421717076-ca4c2f990v3gqt3slc5hjm34ibqdmd27.apps.googleusercontent.com","project_id":"ninth-rain-323420","auth_uri":"https://accounts.google.com/o/oauth2/auth","token_uri":"https://oauth2.googleapis.com/token","auth_provider_x509_cert_url":"https://www.googleapis.com/oauth2/v1/certs","client_secret":"2uNrNCH3H-bMqyyHCktMh1-g","redirect_uris":["http://localhost:8080/"]}}
```

Comment:
I think we should leave this off - committing secrets to GitHub is a no-no right? :p